src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,55 @@
-export const PeopleFilters = () => {
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { getSearchWith } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+
+type Props = {};
+
+export const PeopleFilters: React.FC<Props> = ({}) => {
```

Comment:
The component doesn't receive any props, so there's no need to define an empty `Props` type and use it in the function signature. You can just write `export const PeopleFilters: React.FC = () => {`.